src/Box.js

```diff
@@ -4,11 +4,38 @@ import PropTypes from 'prop-types'
 
 const align = responsiveStyle('text-align', 'align')
 
+
+function boxShadow(props) {
+  const shadowColor = 'rgba(0,0,0,0.08)'
+  const shadows = [
+    `0 0 4px 0 ${shadowColor}`, 
+    `0 8px 8px 0 ${shadowColor}`, 
+    `0 16px 16px 0 ${shadowColor}`, 
+    `0 32px 32px 0 ${shadowColor}`
+  ]
+  const boxShadows = {
+    'small': {
+      'box-shadow': shadows[0]
+    },
+    'medium': {
+      'box-shadow': shadows.slice(0,2).join(", ")
+    },
+    'large': {
+      'box-shadow': shadows.slice(0,3).join(", ")
+    },
+    'xlarge': {
+      'box-shadow': shadows.join(", ")
+    }
+  }
+  return boxShadows[props.boxShadowSize]
+}
+
 const Box = styled.div`
   ${space}
   ${width}
   ${color}
   ${align}
+  ${boxShadow}
```

Comment:
Thanks for putting this together! I think we might want to have the shadow as part of a separate `Card` component, which could extend the Box (e.g. `const Card = styled(Box)`. This would help keep the Box as mostly a layout component, then you could use the Card in places where you want the styles for shadows cc @odmln

Reply:
Sure! I'll create the Card component in a new branch with the changes mentioned in this PR